frontend/packages/console-demo-plugin/src/plugin.tsx

```diff
@@ -11,6 +11,7 @@ import {
   ResourceListPage,
   ResourceDetailPage,
   Perspective,
+  YamlTemplate,
```

Comment:
Just realized this should be YAMLTemplate to match current naming conventions, will update.